sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml

```diff
@@ -29,3 +29,23 @@ robot:
         group_states:
           - home
           - up
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
+  payload_center_of_mass_m: [0.0, 0.0, 0.12]
+
+la_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 0.0
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```

Comment:
No hand

Reply:
There isn't supposed to be I don't think; there's no "hand" element under the "left_manipulator" element, whereas there is under the "right_manipulator" element.